qa/test_framework/util.py

```diff
@@ -459,7 +459,7 @@ def get_coinbase_address(node, expected_utxos=None):
  - the height of the mainchain block at which the sidechain has been created (useful for future checks of mainchain block reference inclusion)
 
 """
-def get_genesis_info(sidechain_id, mainchain_node, withdrawal_epoch_length, secrets, amounts):
+def initialize_new_sidechain_in_mainchain(sidechain_id, mainchain_node, withdrawal_epoch_length, secrets, amounts):
```

Comment:
should receive 1 secret and 1 amount according to future changes of single Creation output with now FT

Reply:
Done